script.py

```
@@ -45,7 +45,7 @@ def _find_available_port(start_port, max_attempts=10):
     for port in range(start_port, start_port + max_attempts):
         try:
             with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
-                s.bind(('127.0.0.1', port))
+                s.bind(('0.0.0.0', port))
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)